beta/plugins/markdownToHtml.js

```diff
@@ -24,6 +26,7 @@ async function markdownToHtml(markdown) {
     .use(images)
     .use(unrwapImages)
     .use(smartyPants)
+    .use(sandpackTargetLanguages)
```

Comment:
I don't understand why the plugins are listed in `remarkPlugins` and composed via `.use`. Just `.use` didn't produce any change.